app/controller/source_data_reader.rb

```diff
@@ -1,4 +1,61 @@
 # frozen_string_literal: true
 
+require 'safe_yaml'
+require 'pry'
+
 class SourceDataReader
+
+  attr_writer :base_path
+
+  def initialize(base_path)
+    @base_path = base_path
+  end
+
+  def create_master_yaml
+    master = { 'topics' => topics, 'conf' => config }
```

Comment:
maybe use symbols instead of strings for all hashes here? 
`topics: topics`
usually I prefer symbols over strings in that case. 

- what's the difference between symbols and strings ?
- when to use strings over symbols?